telemetry/report/utility.py

```diff
@@ -0,0 +1,37 @@
+BOOT_PARTITION_BN_MAP = {
```

Comment:
left is the naming on bootpartition and right is the naming we use in test harness?